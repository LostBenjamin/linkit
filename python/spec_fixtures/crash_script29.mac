

$duplicate-name 5

[duplicate-name 17]
    turquoise

