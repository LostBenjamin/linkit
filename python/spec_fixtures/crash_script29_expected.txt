Macro name 'duplicate-name' is already in use.

